src/game/features/Features.cpp

```diff
@@ -83,8 +91,23 @@ namespace YimMenu
 		});
 	}
 
+	bool TryFirstLoad()
```

Comment:
The function returns bool however I don't see you use it.

Reply:
I was returning a bool in case something needed to be done after. Right now there is no use but I didn't know if it would be needed in the future.

Reply:
Speculative coding is discouraged as you yourself might lose track of it.

Reply:
Okay, it's been changed to a void